user-doc/annotation.md

```diff
@@ -401,12 +402,53 @@ the alternative one has modified the representation of each entity in some way.
- 
+
 See [Context Names](#context-names) section for the list of supported _context_ names. It is assumed that any application context that is performing mutation (record creation, deletion, or editing) MUST use a base entity storage table that is not an abstraction over another table. However, the use of the `detailed` or `compact` context MAY offer an abstraction that augments the presentation of an existing record. An application offering mutation options while displaying an existing entity record might then present the data from the `detailed` or `compact` abstraction but only offer editing or data-entry controls on the fields available from the base storage table.
 
+### 2016 Asset
+
+`tag:isrd.isi.edu,2016:asset`
+
+This key indicates that the annotated table may be interpreted as asset metadata.
+
+Supported JSON payload patterns:
+
+- `null` or `{}`: Default heuristics apply.
+- `{`... `"internal":` [_column_, ...] ...`}`: The one or more named _columns_ store internal identifiers for the asset, used for efficient normalized storage in the database but not meaningful to typical users. The referenced columns MUST each comprise a single-column key for the table.
+- `{`... `"identifier": ` _column_ ...`}`: The _column_ stores a resolvable, globally unique, identifier for the asset. In its simplest form this MAY be satisfied by supplying a URL to the resource. The identifier MUST be _globally unique_ which is satisfiable by any fully qualified HTTP URL, for example. In any case the identifier MUST be _resolvable_ meaning that a client can retrieve the identified asset either directly or indirectly through a resolution service. More formal identifier schemes may also be considered such as MINID, DOI, ARK, or PURL. It is up to client tooling to recognize and resolve identifiers in such schemes.
+- `{`... `"accession_number": ` _column_ ...`}`: The _column_ stores the "accession number" for the asset. The scheme for an accession number is defined by an application (i.e., a science data repository, data hub, community data resource service, etc.). The accession number MUST be guaranteed to be unique _within_ the application. They MAY be globally unique but generally are not, in practice. As an example, accession numbers often take the following form `PREFIX + SERIAL NUMBER [ '.' + REVISION NUMBER ]`, such as `GDS6248`. The example given here is non-normative.
+- `{`... `"name": ` _pattern_ ...`}`: A name for the asset to be derived by a [Pattern Expansion](#pattern-expansion) on _pattern_. The name may be used to represent a file name, but is not required to do so.
+- `{`... `"bytes": ` _column_ ...`}`: The _column_ stores the file size in bytes of the asset. It should be an integer typed column.
+- `{`... `"sha256": ` _column_ ...`}`: The _column_ stores the checksum generated by the 'sha256' cryptographic hash function.
```

Comment:
we should just pick and specify the most common and easy to produce encoding. @mikedarcy in fact, what is the encoding needed by Bags?